rental-components/common/RentalIssueAdvanced.tsx

```diff
@@ -115,7 +115,7 @@ export const RentalIssueAdvanced = ({
               .map((v) => v.type)
               .includes(InvalidationType.Return) ? (
               <Toggle
-                defaultValue={invalidationType === InvalidationType.Reissue}
+                defaultValue={true}
```

Comment:
Now that users have the ability to change a rental to `Return` from `Reissue`, flipping this to true to `Reissue` by default

Reply:
dont think this makes sense, it doesnt align with the logic above which selects Reissue if its rate based by default or Return if its duraiton absed

Reply:
by definition this should default to whether the invalidation type is reissue